waku/waku_archive/driver/sqlite_driver/sqlite_driver.nim

```diff
@@ -66,6 +66,7 @@ method put*(s: SqliteDriver,
   ## Inserts a message into the store
   let res = s.insertStmt.exec((
     @(digest.data),                # id
+    @(digest.data),                # messageHash
```

Comment:
Maybe there is something that I'm missing but `messageHash` looks the same as `id`.

I think we need to create a new `computeDeterministicMessageHash` proc in `waku_archive.nim` where we create the following attribute: https://rfc.vac.dev/spec/14/#deterministic-message-hashing

Reply:
well, as per the initial approach, we aim to remove the `id` attribute and replace it with `messageHash`, as not only nomenclature is better but the way digest/hash is computed it makes sense.

the computation logic behind `messageHash` follows the [RFC](https://rfc.vac.dev/spec/14/#deterministic-message-hashing). The digest logic is implemented in the previous PR for the same base issue #2112

Reply:
You are absolutely right that we have it implemented. I got confused because we have another `computeDigest` definition, which is different. We need to revisit it and check if we can only use one.

https://github.com/waku-org/nwaku/blob/817a7b2e68cdd4e702bb2638531f55f063da6647/waku/waku_store/common.nim#L30